src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -26,7 +29,7 @@ class TerminalLayout extends LayoutBase[ILoggingEvent] {
      */
     val prefix = if (event.getMessage == "{}") s"[${TerminalUtil.highlight(129, "*")}] " else ""
```

Comment:
I'd suggest getting rid of it, I had no idea until now that this was a thing in our logs.  :smile: